Patch-List-Repo/src/main/java/org/wso2/patchvalidator/client/UmtClient.java

```diff
@@ -27,12 +27,12 @@
     private static Properties prop = PropertyLoader.getInstance().prop;
 
     //read patch list using UMT API from UMT DB
-    public static ArrayList<String> getPatchList() {
+    public static ArrayList<String> getPatchList(String state) {
         JSONParser parser = new JSONParser();
         try (CloseableHttpClient httpClient = HttpClientBuilder.create().build()) {
 
             HttpPost request = new HttpPost(prop.getProperty("umtUri"));
-            String jsonInputString = "{'state':'ReadyToSign'}";
+            String jsonInputString = "{'state':'"+ state +"'}";
```

Comment:
```suggestion
            String jsonInputString = "{'state':'" + state + "'}";
```
Please format this line.